homestar-runtime/src/workflow/info.rs

```diff
@@ -330,10 +342,9 @@ impl Info {
     /// workflow [Cid].
     pub(crate) async fn gather<'a>(
```

Comment:
@bgins let's go through `gather` one more time, just want to chat through a couple of things.